docs/deprecations.rst

```diff
@@ -55,6 +55,33 @@ Removed features
 Deprecated features are only removed in major releases after an appropriate
 period of deprecation has passed.
 
+im.offset
+~~~~~~~~~
+
+.. deprecated:: 1.1.2
+.. versionremoved:: 8.0.0
+
+``im.offset()`` has been removed, call ``ImageChops.offset()`` instead.
+
+It was documented as deprecated in PIL 1.1.2,
+raised a ``DeprecationWarning`` since 1.1.5,
+an ``Exception`` since Pillow 3.0.0
+and ``NotImplementedError`` since 3.3.0.
+
+im.fromstring and tostring
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. deprecated:: 2.0.0
+.. versionremoved:: 8.0.0
+
+* ``Image.fromstring()`` has been removed, call ``frombytes()`` instead.
+* ``im.fromstring()`` has been removed, call ``frombytes()`` instead.
+* ``im.tostring()`` has been removed, call ``tobytes()`` instead.
```

Comment:
```suggestion
* ``Image.fromstring()`` has been removed, call ``Image.frombytes()`` instead.
* ``im.fromstring()`` has been removed, call ``im.frombytes()`` instead.
* ``im.tostring()`` has been removed, call ``im.tobytes()`` instead.
```
I'm just slightly concerned that someone would read through your version and think that the two mentions of ``frombytes()`` are referring to the same function.

Reply:
```suggestion
* ``Image.fromstring()`` has been removed, call :py:func:`.Image.frombytes()` instead.
* ``im.fromstring()`` has been removed, call :py:meth:`~PIL.Image.Image.frombytes()` instead.
* ``im.tostring()`` has been removed, call :py:meth:`~PIL.Image.Image.tobytes()` instead.
```

How about providing links instead?
Also need to add `PIL.Image.Image.frombytes` to `Image.rst`, see my other comment.

Reply:
```suggestion
* ``Image.fromstring()`` has been removed, call :py:func:`.Image.frombytes()` instead.
* ``im.fromstring()`` has been removed, call :py:meth:`~PIL.Image.Image.frombytes()` instead.
* ``im.tostring()`` has been removed, call :py:meth:`~PIL.Image.Image.tobytes()` instead.
```

How about also linking to the new functions?
Also need to add documentation for `Image.Image.frombytes`, see my other comment.

*PS: I'm not sure why my comment disappeared before, I may have deleted it accidentally.*